### ISSUE TITLE
resize window when align bottom set

### ISSUE THREAD

**Issue by MarekLew** [Guake/guake#1364]
When I resize  guake window and windows align to bottom of screen then when:
* drag mouse **up** windows resize to **down**
* drag mouse **down** windows resize to **up**
![guake_wrong](https://user-images.githubusercontent.com/6560667/43045804-b1d77006-8dbf-11e8-9d3f-d23230a970b3.gif)

This proble is in `on_resizer_drag` function, at this moment a fix this adding 3 lines:
```
        max_height = screen.get_monitor_geometry(screen_no).height
        + valignment = self.client.get_int(KEY('/general/window_valignment'))
        + if valignment == ALIGN_BOTTOM:
        +   y = max_height - y 
```
My solution is not ideal(mouse point position is tab bar is not at "..."  )
![guake_correct](https://user-images.githubusercontent.com/6560667/43045809-cffd4e20-8dbf-11e8-8b91-76b70d478dfb.gif)

  - **Guake Version**: 0.8.4
  - **system** Linux Mint 18.3 Xfce (64-bits)

**Comment by gsemet** [comment]
thansk for that. Can you do a pull request?

**Comment by MarekLew** [comment]
This is old revision available for Linux Mint 18.3 Xfce. Which branch should I choose? What will be available for this system? He will not scribble the latest one to make sure that it works.

**Comment by aichingm** [comment]
@MarekLew Guake 0.8.x is not supported any more newer versions of distributions will ship with Guake 3.x.x. 

But you always can install Guake from source or via pip. Checkout the documentation at <https://guake.readthedocs.io/en/latest/user/installing.html>
